### PR TITLE
[ReadMe] Just one level/tier of compliance (rebase)

### PR DESCRIPTION
**Pull request by RobDolinMS**
Signed-off-by: Rob Dolin <robdolin@microsoft.com>

### CODE REVIEW

**Comment by RobDolinMS** [comment]
This is a rebase of #543 
/cc @wking @polvi @crosbymichael @mrunalp

File: README.md [hunk 1]
```diff
@@ -24,9 +24,8 @@ Table of Contents
...
 
-An implementation is not compliant for a given CPU architecture if it fails to satisfy one or more of the MUST or REQUIRED requirements for the protocols it implements.
-An implementation is compliant for a given CPU architecture if it satisfies all the MUST and REQUIRED requirements for the protocols it implements.
-An implementation that satisfies all the MUST or REQUIRED and all the SHOULD requirements for its protocols on a given CPU architecture is said to be "unconditionally compliant" with those protocols and architectures.
+An implementation is not compliant for a given CPU architecture if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.
+An implementation is compliant for a given CPU architecture if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.
```
Comment (duglin) [inline README.md]: why are we saying the same thing twice, just backwards?
Also, why do we need to say "CPU architecture"? Why not just "An implementation is compliant if it ...."  ?

Comment (wking) [inline README.md]: On Thu, Sep 01, 2016 at 10:59:01AM -0700, Doug Davis wrote:
> > -An implementation is not compliant for a given CPU architecture if it fails to satisfy one or more of the MUST or REQUIRED requirements for the protocols it implements.
> > -An implementation is compliant for a given CPU architecture if it satisfies all the MUST and REQUIRED requirements for the protocols it implements.
> > -An implementation that satisfies all the MUST or REQUIRED and all the SHOULD requirements for its protocols on a given CPU architecture is said to be "unconditionally compliant" with those protocols and architectures.
> > +An implementation is not compliant for a given CPU architecture if it fails to satisfy one or more of the MUST, REQUIRED, or SHALL requirements for the protocols it implements.
> > +An implementation is compliant for a given CPU architecture if it satisfies all the MUST, REQUIRED, and SHALL requirements for the protocols it implements.
> 
> why are we saying the same thing twice, just backwards?

We could say:

  An implementation is compliant for a given CPU architecture if and only if …

to reduce duplication, but without the “and only if” we need to say it
both ways.

> Also, why do we need to say "CPU architecture"? Why not just "An
> implementation is compliant if it ...."  ?

Compliance is granted for a given architecture (more on this in #527).

**Comment by RobDolinMS** [comment]
@duglin Yes, lines 27 and 28 are similar.

**Comment by RobDolinMS** [comment]
@duglin I kept "CPU architecture" as it was added between my initial commit and this rebase and I wanted this change specifically focused on removing the two tiers/levels of compliance.

**Comment by vbatts** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**